tests/e2e/tools/lib/editor.js

```diff
@@ -79,9 +80,8 @@ const searchAndInsertBlock = async ({page}, blockName, namespace = '') => {
   const addBlockButton = await page.locator(`button[aria-label="${labelText}"]`);
   expect(addBlockButton).toBeVisible();
   await addBlockButton.click();
-
   await page.getByPlaceholder('Search').click();
```

Comment:
Here I would rather use `await page.getByLabel('Search for blocks and patterns').click();` since it's more specific than `Search` which could potentially be used in other inputs on the page.